### ISSUE TITLE
Parse Error

### ISSUE THREAD

**Issue by dejankovacevic** [amimoto-ami/mauticommerce#11]
Dear Developer of Mauticommerce,

I'm glad, that you have published a plugin which is very useful for eCommerce, but i just cannot get your script up and running. I use WP 4.6.1. with latest version of Mautic in cloud.

I am trying to connect those two systems, but im keeping getting error below:

Parse error: syntax error, unexpected '[' in /home/mazu01/domains/enterosgel.si/public_html/wp-content/plugins/mauticommerce/inc/class.order.php on line 73

Could you please help me solve it or give me a hint what to do?

kind regards

**Comment by hideokamoto** [comment]
Hi

Thanks to asking about our plugin.
Cloud you check PHP Version ?
Mauticommerce need php 5.4 or later.

Thanks.

**Comment by dejankovacevic** [comment]
hi,

Thank you for your reply very much. I appreciate it.

Will check.

Kind regards from Europe

Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
w: www.prooptima.si
On 24-Sep-16 1:15:48 PM, Hidetaka Okamoto <notifications@github.com> wrote:
Hi
Thanks to asking about our plugin.
Cloud you check PHP Version ?
Mauticommerce need php 5.4 or later.
Thanks.

**Comment by dejankovacevic** [comment]
Hi,

Since you have developerd PHP script as a plugin fot WP I would like to ask you question about mautic....

Do you have any experiece with sending customized mails from mautic and withing include php code which gets velues from MySQL database...?

Im looking forward to your reply very much!

Thank you

Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
w: www.prooptima.si [http://www.prooptima.si]

On 24-Sep-16 2:21:12 PM, Dejan Kovačevič <kovacevic@prooptima.si> wrote:
hi,

Thank you for your reply very much. I appreciate it.

Will check.

Kind regards from Europe

Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
w: www.prooptima.si
On 24-Sep-16 1:15:48 PM, Hidetaka Okamoto <notifications@github.com> wrote:
Hi
Thanks to asking about our plugin.
Cloud you check PHP Version ?
Mauticommerce need php 5.4 or later.
Thanks.

**Comment by hideokamoto** [comment]
>Do you have any experiece with sending customized mails from mautic and withing include php code which gets velues from MySQL database...?

Unfortunately, I haven't try those things.
Could you ask in mautic's forum?
https://www.mautic.org/community/index.php/conversations/all

thanks.

**Comment by dejankovacevic** [comment]
Thank you for advice. Will ask them.

Enjoy your day


Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
s: dejankovi
w: www.prooptima.si [http://www.prooptima.si]

On 01-Oct-16 2:19:20 PM, Hidetaka Okamoto <notifications@github.com> wrote:
Do you have any experiece with sending customized mails from mautic and withing include php code which gets velues from MySQL database...?
Unfortunately, I haven't try those things.
Could you ask in mautic's forum?
https://www.mautic.org/community/index.php/conversations/all [https://www.mautic.org/community/index.php/conversations/all]
thanks.

**Comment by dejankovacevic** [comment]
Hi there,

Im writing you because of mauticommerce again...i have noticed that all contacts are getting the same ip address (probably IP address od Woocommerce shop),...is there any way to get an IP from client and not from server?

Please find some examples below...

kind regads





Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
s: dejankovi
w: www.prooptima.si [http://www.prooptima.si]

On 02-Oct-16 8:19:23 PM, Dejan Kovačevič <kovacevic@prooptima.si> wrote:
Thank you for advice. Will ask them.

Enjoy your day


Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
s: dejankovi
w: www.prooptima.si [http://www.prooptima.si]

On 01-Oct-16 2:19:20 PM, Hidetaka Okamoto <notifications@github.com> wrote:
Do you have any experiece with sending customized mails from mautic and withing include php code which gets velues from MySQL database...?
Unfortunately, I haven't try those things.
Could you ask in mautic's forum?
https://www.mautic.org/community/index.php/conversations/all [https://www.mautic.org/community/index.php/conversations/all]
thanks.

**Comment by hideokamoto** [comment]
This plugin get IP from `$SERVER` vars.
https://github.com/amimoto-ami/mauticommerce/blob/master/inc/class.order.php#L148

Could you check your infrastructure?
Thanks.

**Comment by dejankovacevic** [comment]
Exactly...but i need IP from client...user who has made a purchase...


Dejan Kovacevic
PRO OPTIMA
a: Stantetova ulica 10, 3320 Velenje
m: +386 31 766 212
e: kovacevic@prooptima.si
s: dejankovi
w: www.prooptima.si [http://www.prooptima.si]

On 20-Oct-16 8:31:12 AM, Hidetaka Okamoto <notifications@github.com> wrote:
This plugin get IP from $SERVER vars.
https://github.com/amimoto-ami/mauticommerce/blob/master/inc/class.order.php#L148 [https://github.com/amimoto-ami/mauticommerce/blob/master/inc/class.order.php#L148]
Could you check your infrastructure?
Thanks.
